compiler/rustc_public/src/unstable/convert/stable/mir.rs

```diff
@@ -252,6 +252,7 @@ impl<'tcx> Stable<'tcx> for mir::Rvalue<'tcx> {
             }
             CopyForDeref(place) => crate::mir::Rvalue::CopyForDeref(place.stable(tables, cx)),
             WrapUnsafeBinder(..) => todo!("FIXME(unsafe_binders):"),
+            StaticallyKnown(..) => todo!(),
```

Comment:
Can I just add this to the public API or is there a different procedure for that?